docs/Compile.rst

```diff
@@ -463,40 +463,102 @@ Building
 
 Windows
 =======
-On Windows, DFHack replaces the SDL library distributed with DF.
+.. contents:: Section Contents
+  :local:
+  :depth: 3
+
+In order to build on windows you'll need to resolve a few dependencies. The dfhack build system uses perl to generate
+source files using xml files detailing various DF structures. Cmake is used to generate build files, or project files
+for visual studio. Read the sections below for more detail.
+
+Note: On Windows, DFHack replaces the SDL library distributed with DF.
 
 Dependencies
 ------------
-You will need the following:
+These are the general dependencies you will need:
 
 * Microsoft Visual C++ 2022, 2019, 2017, or 2015 (optional)
 * Microsoft Visual C++ 2015 Build Tools
 * Git
 * CMake
-* Perl with XML::LibXML and XML::LibXSLT
-
-  * It is recommended to install StrawberryPerl, which includes both.
+* StrawberryPerl
 
+  * Perl
+  * XML:LibXML
+  * XML:LibXLST
 * Python (for documentation; optional, except for release builds)
 
-Microsoft Visual Studio
+Installing Dependencies
 ~~~~~~~~~~~~~~~~~~~~~~~
-Releases of Dwarf Fortress since roughly 2016 have been compiled for Windows using
-Microsoft's Visual Studio 2015 C++ compiler. In order to guarantee ABI and STL compatibility
-with Dwarf Fortress, DFHack has to be compiled with the same compiler.
-
-Visual Studio 2015 is no longer supported by Microsoft and it can be difficult to obtain
-working installers for this product today. As of 2022, the recommended approach
-is to use Visual Studio 2022 or Visual Studio 2019, installing additional optional
-Visual Studio components which provide the required support for using
-Visual Studio 2015's toolchain. All of the required tools are available from Microsoft as part of
-Visual Studio's Community Edition at no charge.
-
-You can also download just the Visual C++ 2015 `build tools`_ if you aren't going to use
-Visual Studio to edit code.
-
-Option 1: Build Tools Only
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. contents:: Section Contents
+  :local:
+  :depth: 1
+
+Chocolatey (`fast install instructions`_) is recommended for installing Git, Cmake, Perl, and Python if they
+are not already installed. Put simply, chocolatey is a package manager for windows so you can install/uninstall
+software from the command line.
+
+.. _fast install instructions: https://chocolatey.org/install
+
+Choco install commands::
+
+   choco install git
+   choco install cmake
+   choco install strawberryperl
+
+There are more dependencies to worry about depending on your intended environment, so read on.
+
+Manually [Fun!]
```

Comment:
The TOC in this section confuses me. Is it presenting "Manually" as the first option, as an alternative to environments 1 & 2? Or is "Manually" an alternative to "install dependencies with chocolatey", and if so, could that have its own subheader?

![image](https://user-images.githubusercontent.com/3719547/174651745-d5a08ac8-dcf6-4298-b587-2ac819acfed1.png)

Reply:
You're right, this is unclear. I can do better. I'll think about how to clarify it.